Cargo.toml

```diff
@@ -52,7 +52,7 @@ ya-utils-networking = "0.2"
 ya-fd-metrics = { path = "utils/fd-metrics" }
 ya-version = "0.2"
 ya-vpn = "0.2"
-ya-client = "0.7.1"
```

Comment:
downgrade?

Reply:
`ya-client` is actually in 0.7.0 right now https://github.com/golemfactory/ya-client/tags

Reply:
And actually it should be 0.7 :)